frappe/utils/pdf.py

```diff
@@ -6,26 +6,26 @@
 from frappe.utils import scrub_urls
 from frappe import _
 from bs4 import BeautifulSoup
-from PyPDF2 import PdfFileReader
-import re
+from PyPDF2 import PdfFileReader, PdfFileWriter
+import re, io
 
 def get_pdf(html, options=None, output = None):
 	html = scrub_urls(html)
 	html, options = prepare_options(html, options)
-	fname = os.path.join("/tmp", "frappe-pdf-{0}.pdf".format(frappe.generate_hash()))
+	# fname = os.path.join("/tmp", "frappe-pdf-{0}.pdf".format(frappe.generate_hash()))
```

Comment:
Why is this commented out?

Reply:
We don't need fname anymore. Removed the comment all together